### PR TITLE
Try to improve headers colors

### PR DESCRIPTION
**Pull request by MangelMaxime**
Related: #3 

I used existing colors from Bulma and try to find one that match with the navbar and the dark theme. If you want to experiment by yourself, you can see the changes in the diff

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-let fableBranch = "dev2.0"
+let fableBranch = "dev2.0-trampoline"
```
Comment (alfonsogarciacaro) [inline build.fsx]: Can you please undo this before merging?

**Comment by MangelMaxime** [comment]
@alfonsogarciacaro Done

**Comment by ncave** [comment]
@MangelMaxime Looks good, what about the toolbar/menubar, is that feasible?

**Comment by MangelMaxime** [comment]
> is that feasible?

Yes it is, but I am really not sure of the result. And if the goal, in the end is to have a global dark theme, I will probably customize Bulma framework directly so it auto apply everything. I will give it a try and share my result.

**Comment by alfonsogarciacaro** [comment]
[Bulmaswatch](https://jenil.github.io/bulmaswatch/slate/) was quite useful in another project.

**Comment by MangelMaxime** [comment]
I am creating a theme based on the `vs-dark` theme.

I am doing that so both the editor and the "website" share the same colors. So, in theory we will have something related to how VSCode looks with `vs-dark` theme.

**Comment by MangelMaxime** [comment]
Bonus point, is that in future because we know which variable to set/customize we could probably offer a `vs-light`, etc. version

**Comment by MangelMaxime** [comment]
Closed in favor of #7

**Pull request closed without merging**